rc_simulator/examples/sunAngles.py

```diff
@@ -18,20 +18,19 @@
 
 
 def calculate_sun_angles():
-    Zurich = Location(epwfile_path=os.path.join(
+    Zurich = Location(path=os.path.join(
```

Comment:
I would still prefer this attribute to be called epfile_path as path can be confused with other potential attributes